pkg/util/testing/wrappers.go

```diff
@@ -143,6 +151,15 @@ func (w *WorkloadWrapper) Condition(condition metav1.Condition) *WorkloadWrapper
 	return w
 }
 
+func (w *WorkloadWrapper) SetOrReplaceCondition(condition metav1.Condition) *WorkloadWrapper {
```

Comment:
`apimeta.SetStatusCondition` already implements this.

Reply:
I use it here https://github.com/kubernetes-sigs/kueue/pull/717/files#diff-2983f8941450021febb0b5013e359a5f344a84a6f6a9cfee8ecbdf3c3f92a9cfR756

I need to replace LastTransitionTime from condition but apimeta.SetStatusCondition doesn't do that if !newCondition.LastTransitionTime.IsZero().

Reply:
I see, because `Admit` adds the condition...

Sounds good.